src/web/components/ParticipantManagement/AddParticipantDialog.tsx

```diff
@@ -78,28 +77,17 @@ function AddParticipantDialog({
       }
     });
     return () => subscription.unsubscribe();
-  }, [watch, setValue, open]);
-
-  useEffect(() => {
-    if (!open) {
-      // the dialog doesn't de-render on close, so we need to clean up our state
-      setSearchText('');
-      setSiteSearchResults(undefined);
-      setSelectedSite(undefined);
-      setNewSite(false);
-      reset();
-    }
-  }, [open, reset]);
```

Comment:
Can you please verify I removed the right useEffect() but also that the other two useEffect() functions should stay?

Reply:
LGTM